Makefile.am

```diff
@@ -181,6 +189,9 @@
 		--define="vdsm_release $(PACKAGE_RELEASE)" \
 		$(DIST_ARCHIVES)
 
+clean-build:
```

Comment:
Why not to put it as a dependency also to `clean` target?

Reply:
There is no clean target in the Makefile, if I understood correctly the flow, it is generated when you call the `autogen.sh` script. The way to influence which **files** will be removed in the `clean` target is by adding them to the `CLEANFILES` set.

Reply:
It seems `clean-local` target can be used for the purpose: https://www.gnu.org/software/automake/manual/automake.html#Clean

Reply:
Great!

Atm I could just change `clean-build` ->  `clean-local` and it would work, but in case we want to extend it, I think it would be nice to keep them separated.

So I will keep `clean-build` and create a `clean-local` that depends on `clean-build`.

Reply:
Changing clean-build to clean-local will make clean-local run when running `make clean`.
This is nice, but you already added the files in the build directory to CLEANFILES list,
so they are already cleaned when running `make clean`.

However when this is called `clean-local` people will add more files there and since
we depend on this target in `make rpm` these files will be remove, even if they are 
needed for building rpms.

Did you test

   make srpm rpm

works and create both rpms and srpm? Or does it create the first target and then remove
the build directory before creating the next?

Reply:
On your first point, is more or less what I tried to say above. If we want to have a clean-local target for the future and keep adding other folders, having `clean-build` and `clean-local` separated is the right approach. Also, that allows to construct `clean-local` with dependencies if we want. And I can remove the wildcard from the `CLEANFILES` set.

About the
   
    make srpm rpm

That's a good point that I didn't think about, but I just tested and works fine, it only cleans it once at first.